components/Allocations/SelectWorker/SelectWorker.tsx

```diff
@@ -0,0 +1,84 @@
+import cx from 'classnames';
+import { Worker } from 'types';
+
+const ResultEntry = ({
+  worker,
+  callback,
+}: {
+  worker: Worker;
+  callback: any;
+}): React.ReactElement => {
+  const { id, firstName, lastName, allocationCount } = worker;
+
+  return (
+    <>
+      <tr className={cx('govuk-table__row')}>
+        <td className="govuk-table__cell">
+          <div className="govuk-radios__item">
+            <input
+              id={String(id)}
+              name="workerId"
+              type="radio"
+              value={id}
+              onChange={(e) => {
+                callback(e.target.value);
+              }}
+              className={`govuk-radios__input`}
+            />
+            <label
+              className="govuk-label govuk-radios__label"
+              htmlFor={String(id)}
+            >
+              <span className="govuk-visually-hidden">
+                {`${firstName} ${lastName}`}
+              </span>
+            </label>
+          </div>
+        </td>
+        <td data-testid="workerName" className="govuk-table__cell">
+          {`${firstName} ${lastName}`}
+        </td>
+        <td className="govuk-table__cell">
+          <label data-testid="allocationCount" htmlFor={String(id)}>
+            {allocationCount}
+          </label>
+        </td>
+      </tr>
+    </>
+  );
+};
+
+const SelectWorker = ({
```

Comment:
Little tip here, not something you must update, but you can make typing functions a lot nicer like this:

```typescript
const SelectWorker: React.FC<{
  records: Worker[];
  callback: (id: string) => void;
}> = ({ records, callback }) => (
  <table className="govuk-table lbh-table" data-testid="residents-table">
```

Also, I'd just specify the function signature rather than passing `any`.

Reply:
Added function signature